Dockerfile

```diff
@@ -37,6 +44,9 @@ WORKDIR /kvrocks
 
 COPY --from=build /kvrocks/build/kvrocks ./bin/
 
+COPY --from=build /kvrocks/tools/redis-cli ./bin/
+RUN ln -s /kvrocks/bin/redis-cli /usr/bin/redis-cli
```

Comment:
Is it necessary to add this symlink?

Reply:
/kvrocks/bin is not in the PATH environment variable, so I created a symlink. Are you suggesting that I should either copy it directly to /usr/bin or add /kvrocks/bin to the PATH?

Reply:
I think the latter is better.